### ISSUE TITLE
waypoint server install [ECS] can't accept a list/array of subnets 

### ISSUE THREAD

**Issue by mreferre** [hashicorp/waypoint#1651]
**Describe the bug**
It is not possible to specify a list of subnets in the `waypoint server install` to the `ECS` platform. By default Waypoint pick all the subnets of the default VPC. However, as a user I would like to be able to explicitly pick a number of subnets either in the same VPC or another VPC.    

**Steps to Reproduce**
I am trying to run the following command and the shell keeps complaining that the `["subnet-d22cbcb4","subnet-0ad86842"]` subnet does not exist. The two subnets are two subnets of the default VPC.

```
waypoint server install -accept-tos \
        -platform=ecs \
        -ecs-cluster=waypoint-server \
        -ecs-cpu=512 \
        -ecs-mem=1024 \
        -ecs-execution-role-name=waypoint-server-execution-role \
        -ecs-region=us-west-2 \
        -ecs-server-image=hashicorp/waypoint:latest \
        -ecs-subnets=["subnet-d22cbcb4","subnet-0ad86842"]
``` 
It seems to be possible to pick a single subnet (by using `-ecs-subnets="subnet-d22cbcb4"`) but it is not possible to pick 2+ subnets. In other words it doesn't seem to be possible to pass a list/array of subnets.  

**Expected behavior**
I'd expect the server install command to pick a list/array of subnets and not just a single subnet.   

**Waypoint Platform Versions**
Additional version and platform information to help triage the issue if
applicable:

* Waypoint CLI Version: 0.4.0 
* Waypoint Server Platform and Version: `ECS`
* Waypoint Plugin: not applicable

**Additional context**
It is unclear to me if this is a syntax issue (I have tried a number of variants of the syntax above to no avail) or if it's a bug.

**Comment by catsby** [comment]
Hey @mreferre - apologies for the confusion here, I believe the correct syntax is `-ecs-subnets=subnet-d22cbcb4,subnet-0ad86842`, could you give that a try?

**Comment by mreferre** [comment]
Thanks @catsby for the quick turn around. Nope, I think that is one of the many variations I have tried. Just gave it a try again: 

```
sh-3.2$ waypoint server install -accept-tos \
>         -platform=ecs \
>         -ecs-cluster=waypoint-server \
>         -ecs-cpu=512 \
>         -ecs-mem=1024 \
>         -ecs-execution-role-name=waypoint-server-execution-role \
>         -ecs-region=us-west-2 \
>         -ecs-server-image=hashicorp/waypoint:latest \
>         -ecs-subnets=subnet-d22cbcb4,subnet-0ad86842
 ! Using provided subnets for Service networking
! Error installing server into ecs: InvalidSubnetID.NotFound: The subnet ID 'subnet-d22cbcb4,subnet-0ad86842' does
  not exist
  	status code: 400, request id: 32ad6d4b-4b5d-476a-8a4c-14bca7e1e082
sh-3.2$
``` 

It just treats it the string as a single subnet (not as a list).

**Comment by catsby** [comment]
hrm very strange, I'll dig in further thanks!

**Comment by catsby** [comment]
Hey @mreferre thanks again for reporting this. I fixed it in #1669 but updating how we parse the inputs. After the next release, the correct format will be what I suggested above `-ecs-subnets=subnet1,subnet2`
